spec/virtual_includes_spec.rb

```diff
@@ -224,7 +224,10 @@
   it "preloads virtual_attribute in :include when :conditions are also present in calculations" do
     expect(Book.includes([:author_name, :author]).references(:author).where("authors.name = '#{author_name}'")).to preload_values(:author_name, author_name)
     expect(Book.includes([:author_name, :author]).references(:author).where(:authors => {:name => author_name})).to preload_values(:author_name, author_name)
+    # specifically, we are specifically testing this use case
```

Comment:
redundant wording...maybe
```suggestion
    # Disable Rails/WhereNot because we are testing this specific use case
```